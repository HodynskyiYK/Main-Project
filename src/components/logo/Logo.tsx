import React, { FC } from 'react'
import styles from './Logo.module.scss'

export const Logo: FC = () => {

    return (
        <span className={styles.Logo}>
            <b>netflix</b>
            roulette
        </span>
    )
}
