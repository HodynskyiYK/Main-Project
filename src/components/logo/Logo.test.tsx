import React from 'react'
import { Logo } from './Logo'
import { render } from '@testing-library/react'


describe('Logo tests', () => {

    test('To match snapshot', () => {
        const {asFragment} = render(<Logo/>)
        expect(asFragment()).toMatchSnapshot()
    })
})