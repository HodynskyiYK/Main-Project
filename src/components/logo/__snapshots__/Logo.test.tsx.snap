// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo tests To match snapshot 1`] = `
<DocumentFragment>
  <span
    class="Logo"
  >
    <b>
      netflix
    </b>
    roulette
  </span>
</DocumentFragment>
`;
